common.blocks/button/button.bemhtml

```diff
@@ -1,4 +1,11 @@
 block('button')(
+    def()(function() {
+        var isRealButton = (['button', 'input'].indexOf(apply('tag')) >= 0) &&
```

Comment:
We have a `NOTE` comment in the `content` mode below about mode call optimisations. But here we call `tag` which will be called again within `applyNext()`. Maybe we should be more strict?